### PR TITLE
Beaker DOS fix

### PR DESCRIPTION
**Pull request by dirkgr**

### CODE REVIEW

**Review by epwalsh — APPROVED** [review]

File: tango/integrations/beaker/workspace.py [hunk 1]
```diff
@@ -292,7 +295,10 @@ def _get_run_from_dataset(self, dataset: Dataset) -> Optional[Run]:
 
         import concurrent.futures
 
-        with concurrent.futures.ThreadPoolExecutor() as executor:
+        with concurrent.futures.ThreadPoolExecutor(
+            max_workers=50,
```
Comment (epwalsh) [inline tango/integrations/beaker/workspace.py:299]: Actually this probably makes the situation worse. The default is not more than 36.
https://github.com/python/cpython/blob/e929dae42ad6aaf55e5cce6339c1ef97c01c8f7b/Lib/concurrent/futures/thread.py#L142

**Review by epwalsh — CHANGES_REQUESTED** [review]

File: tango/integrations/beaker/workspace.py (hunk 1 above)
Comment (dirkgr) [reply tango/integrations/beaker/workspace.py:299]: Oh. Then the documentation for that is wrong.

Comment (epwalsh) [reply tango/integrations/beaker/workspace.py:299]: Yeup 🙃

**Comment by dirkgr** [comment]
@epwalsh good to go?

**Comment by epwalsh** [comment]
Can we make max workers a configurable parameter so this isn't always the default? Or we just need to remember to roll this back once it no longer breaks Beaker.

**Comment by dirkgr** [comment]
Since the limit is pretty much a function of the beaker service, I think it should be hardcoded, just like other aspects of the beaker service are hardcoded.

**Comment by epwalsh** [comment]
Fair enough. Can we at least make an issue to roll it back once fixed so we don't forget?

**Review by epwalsh — APPROVED** [review]

**Comment by dirkgr** [comment]
Tracking in https://github.com/allenai/tango/issues/317.

**Pull request merged**